src/tests/ftest/launch.py

```diff
@@ -878,14 +878,21 @@ def run_tests(test_files, tag_filter, args):
 
             # Optionally rename the test results directory for this test
             if args.rename:
-                rename_logs(avocado_logs_dir, test_file["py"])
+                rename_logs(avocado_logs_dir, test_file["py"], args)
 
             # Optionally process core files
             if args.process_cores:
                 if not process_the_cores(avocado_logs_dir, test_file["yaml"],
                                          args):
                     return_code |= 256
 
+    if args.jenkinslog:
+        test_logs_lnk = os.path.join(avocado_logs_dir, "latest")
+        try:
+            os.remove(test_logs_lnk)
+        except OSError as error:
+            print("Failed to remove sym link latest: {}".format(error))
+
```

Comment:
I think there may be a use case to run launch.py locally with the new `-j` option, where the user would still want the `avocado/job-results/latest` to exist.  I also think the removal of the `avocado/job-results/latest` directory is better handled in `ci/functional/job_cleanup.sh` as it does currently.